src/api/app/models/staging/workflow.rb

```diff
@@ -101,11 +114,11 @@ def update_staging_projects_managers_group
     # FIXME: This assignation is need because after store a staging_project
     # the object is reloaded and we lost the changes.
     self.managers_group = new_managers_group
-  end
 
-  def add_reviewer_group
-    role = Role.find_by_title('reviewer')
-    project.relationships.find_or_create_by(group: managers_group, role: role)
+    # update reviewer group in staging project
```

Comment:
the comment is wrong. this is not about the staging project but the project the workflow is for (do you have a name for that? We call it normally target project)

Reply:
@coolo:

> the comment is wrong. this is not about the staging project but the project the workflow is for (do you have a name for that? We call it normally target project)

Right. Based in our [wiki documentation](https://github.com/openSUSE/open-build-service/wiki/Staging-Workflow) I will rename it to "staging workflow project".